### PR TITLE
add FAIR software checklist to recommendations

### PR DESCRIPTION
**Pull request by egpbos**
@jspaaks together with ARDC created https://fairsoftwarechecklist.net. I think it is an ideal quality checklist for research software (better than the CII one that's currently on top, which is too software-oriented and too detailed as well for most projects) and we should promote it whereever we can, including here, if you agree.

### CODE REVIEW

**Comment by c-martinez** [comment]
Netlify deployed @egpbos , does it look as you expected?

File: static/json/recommendations.json [hunk 1]
```diff
@@ -253,7 +253,7 @@
...
...: "<ul><li><a target='_blank' href='https://fairsoftwarechecklist.net'></a>FAIR software checklist (online, interactive)</li><li><a target='_blank' href='https://bestpractices.coreinfrastructure.org/en'>Core Infrastructures Initiative</a> (online, interactive)</li><li>Deutsches Zentrum für Luft- und Raumfahrt <a target='_blank' href='https://rse.dlr.de/download/checklist_applicationclass_1-markdown_v1.0.md'>Class 1</a>, <a target='_blank' href='https://rse.dlr.de/download/checklist_applicationclass_2-markdown_v1.0.md'>Class 2</a>, <a target='_blank' href='https://rse.dlr.de/download/checklist_applicationclass_3-markdown_v1.0.md'>Class 3</a> (MarkDown)</li><li>Software Sustainability Institute’s software evaluation checklist (<a href='https://docs.google.com/forms/d/e/1FAIpQLSf0ccsVdN-nXJCHLluJ-hANZlp8rDKgprJa0oTYiLZSDxh3DA/viewform' target='_blank'>Google form</a>)</li><li>CLARIAH checklist (<a href='https://github.com/CLARIAH/software-quality-guidelines/blob/v1.0/softwareguidelines.pdf' target='_blank'>PDF page 38-42</a>)</li><li>EURISE (<a href='https://github.com/eurise-network/technical-reference/blob/v0.1/quality/software-checklist.rst' target='_blank'>MarkDown</a>)</li></ul>"
```
Comment (c-martinez) [inline static/json/recommendations.json:256]:
```suggestion
                        "text" : "<ul><li><a target='_blank' href='https://fairsoftwarechecklist.net'>FAIR software checklist </a>(online, interactive)</li><li><a target='_blank' href='https://bestpractices.coreinfrastructure.org/en'>Core Infrastructures Initiative</a> (online, interactive)</li><li>Deutsches Zentrum für Luft- und Raumfahrt <a target='_blank' href='https://rse.dlr.de/download/checklist_applicationclass_1-markdown_v1.0.md'>Class 1</a>, <a target='_blank' href='https://rse.dlr.de/download/checklist_applicationclass_2-markdown_v1.0.md'>Class 2</a>, <a target='_blank' href='https://rse.dlr.de/download/checklist_applicationclass_3-markdown_v1.0.md'>Class 3</a> (MarkDown)</li><li>Software Sustainability Institute’s software evaluation checklist (<a href='https://docs.google.com/forms/d/e/1FAIpQLSf0ccsVdN-nXJCHLluJ-hANZlp8rDKgprJa0oTYiLZSDxh3DA/viewform' target='_blank'>Google form</a>)</li><li>CLARIAH checklist (<a href='https://github.com/CLARIAH/software-quality-guidelines/blob/v1.0/softwareguidelines.pdf' target='_blank'>PDF page 38-42</a>)</li><li>EURISE (<a href='https://github.com/eurise-network/technical-reference/blob/v0.1/quality/software-checklist.rst' target='_blank'>MarkDown</a>)</li></ul>"
```
Closed the </a> too early ;-)

Comment (egpbos) [reply static/json/recommendations.json:256]: Ah, thanks! Hard to see :)

**Comment by egpbos** [comment]
Indeed, the link is wrong in the Netlify deployment. Waiting for the new deploy...

**Comment by egpbos** [comment]
Ah yes, it looks right now! Oh, I actually see now that there is also text that mentions the CII one explicitly as "one of our favorites". I would personally suggest we change that too, but this may be more contentious. With whom should we discuss that?

**Comment by egpbos** [comment]
Do we also need to check with someone at DANS or are they no longer involved?

**Pull request merged**